manifests/instance.pp

```diff
@@ -276,7 +277,11 @@
     refreshonly => true,
   }
 
-  $redis_version_real = pick(getvar_emptystring('redis_server_version'), $minimum_version)
+  if $package_ensure =~ /^[0-9]\.[0-9]/ {
```

Comment:
Hmm, we might want to extend this regex. If someone's using an rc/alpha candidate this regex won't match (eg. https://github.com/antirez/redis/archive/4.0-rc3.tar.gz)

Can we extend the regex and add some extra usecases to the specs? eg. 3 digits (3.2.1), non-numeric characters (4.0.0-rc)